### PR TITLE
[ONNX] add sigmoid and squeeze operator

### PR DESCRIPTION
**Pull request by Herald-of-Aurora**

### CODE REVIEW

**Comment by Herald-of-Aurora** [comment]
Jenkins passed, id: 1132. (/view/ZOO-PR/job/ZOO-PR-Validation/1132/)

File: zoo/src/main/scala/com/intel/analytics/zoo/pipeline/api/autograd/math.scala [hunk 1]
```diff
@@ -460,8 +460,22 @@ class Variable[T: ClassTag] private[zoo] (private[zoo] var node: ModuleNode[T],
...
-    Variable(layer.inputs(this.node))
+    val dims = new Array[Int](1)
+    dims(0) = dim
+    squeeze(dims)
+  }
+
+  def squeeze(dims: Array[Int]): Variable[T] = {
+    if (dims == null){
+      val blayer = com.intel.analytics.bigdl.nn.Squeeze[T](null, batchMode = false)
+      val klayer = new KerasLayerWrapper[T](blayer)
```
Comment (zhichao-li) [inline zoo/src/main/scala/com/intel/analytics/zoo/pipeline/api/autograd/math.scala]: move these two lines out of the condition check:
```
      val klayer = new KerasLayerWrapper[T](blayer)
      Variable(klayer.inputs(this.node))
```

File: pyzoo/test/zoo/pipeline/onnx/test_model_loading.py [hunk 2]
```diff
@@ -480,3 +480,25 @@ def test_sub(self):
...
+
+    def test_squeeze(self):
+        node = onnx.helper.make_node(
+            'Squeeze',
+            inputs=['x'],
+            outputs=['y'],
+            axes=[0],
+        )
+        x = np.random.randn(1, 3, 4, 5).astype(np.float32)
+        y = np.squeeze(x, axis=0)
```
Comment (zhichao-li) [inline pyzoo/test/zoo/pipeline/onnx/test_model_loading.py]: add a unitest when axis = 1

File: pyzoo/zoo/pipeline/api/onnx/mapper/sigmoid.py [hunk 3]
```diff
@@ -0,0 +1,27 @@
+#
+# Copyright 2018 Analytics Zoo Authors.
+#
```
Comment (zhichao-li) [inline pyzoo/zoo/pipeline/api/onnx/mapper/sigmoid.py]: any unitest for sigmoid?

**Comment by Herald-of-Aurora** [comment]
Jenkins passed, id: 1133. (/job/ZOO-PR-Validation/1133/)

**Pull request merged**